jobbergate_api/main.py

```diff
@@ -0,0 +1,18 @@
+from fastapi import FastAPI
+from gino.ext.starlette import Gino
+
+from .config import settings
+from .routers import load_routers
+
+db = Gino()
+
+
+def get_app(db, database_url):
```

Comment:
you should separate app creation and db initialization for testability and to make it easier to refactor.

`fastapi` actually has event routers that are extremely useful for this kind of thing, see [init_database here](https://github.com/omnivector-solutions/license-manager/blob/main/src/licensemanager2/backend/main.py#L68)

Reply:
you might want to steal the `handler` function from the same file, because you'll need it in a lambda context, especially for api gateway